lib/features/exhibition_flow/widgets/organisation_detail_bottomsheet.dart

```diff
@@ -0,0 +1,122 @@
+            topLeft: Radius.circular(10), topRight: Radius.circular(10)),
+        child: Scaffold(
+          extendBodyBehindAppBar: true,
+          appBar: AppBar(
+            forceMaterialTransparency: true,
+            automaticallyImplyLeading: false,
+            actions: [
+              IconButton(
+                onPressed: () => context.pop(),
+                padding: const EdgeInsets.only(right: 16),
+                icon: SvgPicture.asset(
+                  'assets/images/close_icon.svg',
+                  alignment: Alignment.centerRight,
+                ),
+              )
+            ],
+          ),
+          body: Scrollable(
+            viewportBuilder: (context, offset) => ListView(
+              children: [
+                OrganisationHeader(
+                  organisation: organisation,
+                ),
+                Container(
+                  height: 168,
+                  width: double.maxFinite,
+                  margin:
+                      const EdgeInsets.symmetric(vertical: 12, horizontal: 24),
+                  child: Image.network(
+                    organisation.promoPictureUrl,
+                    fit: BoxFit.cover,
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(20, 0, 20, 100),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text(
+                        organisation.name,
+                        textAlign: TextAlign.start,
+                        style:
+                            AppTheme.lightTheme.textTheme.titleLarge?.copyWith(
+                          fontWeight: FontWeight.bold,
+                          fontSize: 20,
+                        ),
+                      ),
+                      const SizedBox(height: 12),
+                      Text(
+                        organisation.shortDescription,
+                        textAlign: TextAlign.start,
+                        style:
+                            AppTheme.lightTheme.textTheme.titleMedium?.copyWith(
+                          fontWeight: FontWeight.w600,
+                          fontSize: 16,
+                        ),
+                      ),
+                      const SizedBox(height: 12),
+                      Text(
+                        organisation.longDescription,
+                        textAlign: TextAlign.start,
+                      ),
+                      const SizedBox(height: 12),
+                      Text(
+                        "Even \$1 helps!",
+                        textAlign: TextAlign.start,
+                        style:
+                            AppTheme.lightTheme.textTheme.labelMedium?.copyWith(
+                          color: AppTheme.givt4KidsBlue,
+                          fontWeight: FontWeight.bold,
+                          fontSize: 20,
```

Comment:
16px according to Figma

Reply:
actually I don't think we need to override font sizes at all if we use themes and topography. 
But, since redesigning existing components like this one from recommendation flow was out of scope of the story, +   this entire exhibition_flow folder is temporary I decided to skip such parts of code for now. 
@MaikelStuivenberg What do you think?